hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/StorageLocationReport.java

```diff
@@ -227,34 +232,28 @@ public static StorageLocationReport getFromProtobuf(StorageReportProto report)
     return builder.build();
   }
 
-  /**
-   * Returns the StorageLocationReport from the protoBuf message.
-   * @param report MetadataStorageReportProto
-   * @return StorageLocationReport
-   * @throws IOException in case of invalid storage type
-   */
-
-  public static StorageLocationReport getMetadataFromProtobuf(
```

Comment:
This method is also being removed by #8208, so I'll wait for its merge before updating the PR.